core/ai_worker.go

```diff
@@ -0,0 +1,1054 @@
+package core
+
+import (
+	"bytes"
+	"context"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"os"
+	"path"
+	"strconv"
+	"sync"
+	"time"
+
+	"github.com/ethereum/go-ethereum/crypto"
+	"github.com/golang/glog"
+	"github.com/livepeer/ai-worker/worker"
+	"github.com/livepeer/go-livepeer/clog"
+	"github.com/livepeer/go-livepeer/common"
+	"github.com/livepeer/go-livepeer/monitor"
+	"github.com/livepeer/go-livepeer/net"
+	"github.com/livepeer/go-tools/drivers"
+	"github.com/livepeer/lpms/ffmpeg"
+)
+
+var ErrRemoteWorkerTimeout = errors.New("Remote worker took too long")
+var ErrNoCompatibleWorkersAvailable = errors.New("no workers can process job requested")
```

Comment:
Why is this exported ?

Reply:
Followed how other similar errors are implemented in core package. Linked errors and errors above could probably be changed to not be exported.

https://github.com/livepeer/go-livepeer/blob/4a66b2232188d765a7d04bcc1f5ed21a47ef67ef/core/orchestrator.go#L852